doc/reference/reference_lua/box_stat/memtx.rst

```diff
@@ -0,0 +1,181 @@
+    * ``avg`` is the average number of bytes that a single transaction uses
+      (equals ``total`` / number of open transactions).
+    * ``max`` is the maximal number of bytes that a single transaction uses.
+
+* ``box.stat.memtx().tx.mvcc`` shows memory allocation related to
+  :ref:`multiversion concurrency control (MVCC) <txn_mode_transaction-manager>`.
+  MVCC is reponsible for isolating transactions.
+  It reveals conflicts and makes sure that tuples that do not belong to a particular
+  space but were (or could be) read by some transaction were not deleted.
+
+  It consists of the following sections:
+
+  * ``trackers`` is the memory allocated for *trackers* of transaction reads.
+    Like in the :ref:`previous sections <box_introspection-box_stat_memtx_tx_total_avg_max>`,
+    Tarantool reports the total, average, and maximal number of bytes allocated
+    for trackers per a single transaction.
+  * ``conflicts`` is the memory allocated for *conflicts*
+    which are entities created when transactional conflicts occur.
+    Like in the :ref:`previous sections <box_introspection-box_stat_memtx_tx_total_avg_max>`,
+    Tarantool reports the total, average, and maximal number of allocated bytes.
+  * ``tuples`` is the memory allocated for storing tuples.
+    With MVCC, tuples are stored using the *stories* mechanism. Nearly every
+    tuple has its story. Even tuples in an index may have their stories, so
+    it may be useful to differentiate memory allocated for tuples and memory
+    allocated for stories.
+
+    All stored tuples fall into three categories, with memory statistics
+    reported for each category:
+
+    * ``tracking`` is for tuples that are not used by any transactions directly,
+      but MVCC uses them for tracking transaction reads.
+    * ``used`` is for tuples that are used by active read-write transactions.
+      See a detailed :ref:`example <box_introspection-box_stat_memtx_tx_example>` below.
+    * ``read_view`` is for tuples that are not used by active read-write transactions,
+      but are used by read-only transactions.
+
+      For each of the three categories, Tarantool reports two statistical blocks:
+
+      * ``stories`` is for stories.
+      * ``retained`` is for *retained* tuples which do not belong to any index,
+        but MVCC doesn't allow to delete them yet.
+
+      For each block, Tarantool reports the following statistics:
+
+      * ``count`` is the number of stories or retained tuples.
+      * ``total`` is the number of bytes allocated for stories or retained tuples.
+
+.. _box_introspection-box_stat_memtx_tx_example:
+
+**Example**
+
+Let's get memory statistics for ``used`` tuples in a transaction.
+
+First, we :ref:`enable MVCC <txn_mode_mvcc-enabling>` so that
+``box.stat.memtx.tx().mvcc`` contained non-zero values.
+
+.. code-block:: lua
+
+   box.cfg{memtx_use_mvcc_engine = true}
+
+We did it within the first ``box.cfg{}`` call to a new Tarantool instance,
```

Comment:
`box.cfg` is deprecated since it's dangerous when mixed with YAML config. We avoid creating new content with this approach. I'd just say that the cluster must be started with [database.use_mvcc_engine](https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/#confval-database.use_mvcc_engine) set to `true`.